tests/valid_nim_functions.stoffel

```diff
@@ -0,0 +1,64 @@
+# Basic Nim-style function definition
+proc add(a: int, b: int): int =
```

Comment:
I'm being a stickler but the syntax should be closer to Python than to nim. Way more people know Python and it's basically how it's been marketed so far.

Reply:
Frankly, to me, a pure function just warrants a keyword and not a whole new function name to declare it. The usual python way would be to add a decorator, which I agree probably isn't the way to go.